src/test/external_dependencies_test.ts

```diff
@@ -1,50 +1,40 @@
 
-import * as test    from "https://deno.land/std@0.208.0/assert/mod.ts";
-import * as path    from "https://deno.land/std@0.208.0/path/mod.ts";
-import * as infer   from '../infer.ts';
+import * as test  from "https://deno.land/std@0.208.0/assert/mod.ts";
+import * as path  from "https://deno.land/std@0.208.0/path/mod.ts";
+import * as infer from '../infer.ts';
 
 // Skipped due to NPM dependency resolution not currently being supported.
-Deno.test({
-  name: "Inference",
-  fn() {
-    const program_path = path.fromFileUrl(import.meta.resolve('./data/external_dependencies.ts'));
-    const vendor_path = path.fromFileUrl(import.meta.resolve('./vendor'));
-    const program_results = infer.programInfo(program_path, vendor_path, true);
+Deno.test("External Dependencies", () => {
+  const program_path = path.fromFileUrl(import.meta.resolve('./data/external_dependencies.ts'));
+  const vendor_path = path.fromFileUrl(import.meta.resolve('./vendor'));
+  const program_schema = infer.inferProgramSchema(program_path, vendor_path, true);
 
-    test.assertEquals(program_results, {
-      positions: {
-        test_deps: [
-          "s"
-        ]
-      },
-      schema: {
-        scalar_types: {
-          String: {
-            aggregate_functions: {},
-            comparison_operators: {},
-          },
-        },
-        object_types: {},
-        collections: [],
-        functions: [],
-        procedures: [
+  test.assertEquals(program_schema, {
+    scalar_types: {
+      String: {},
```

Comment:
Do we ever put anything in the scalar values?

Reply:
Not for now; it's just an empty object.

Reply:
Not that it really matters and this might help with future proofing, but could just keep an array of names if we wanted to use the simplest rep.

Reply:
The nice thing about the map is that it guarantees name uniqueness structurally, but that's minor.

Reply:
That's true. No point bikeshedding let's keep it!